xdsl/irdl.py

```diff
@@ -755,16 +939,31 @@ def from_pyrdl(pyrdl_def: type[_OpT]) -> OpDef:
                 continue
             clsdict = {**clsdict, **parent_cls.__dict__}
```

Comment:
Seems unused now!

Reply:
Actually this one is still used, but I deleted one on line 1509